### PR TITLE
For #1622: BatchClaims fake implementation.

### PR DESCRIPTION
**Pull request by paulodamaso**
For #1622: BatchClaims fake implementation.
- Added `FkBatchClaims`, a decorator to `BatchClaims` testing
- Corrected test in `BatchClaimsTest` to use `FkBatchClaims`

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1632](https://github.com/zerocracy/farm/issues/1632) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1632](https://github.com/zerocracy/farm/issues/1632) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/2340626752); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/claims/BatchClaimsTest.java [hunk 1]
```diff
@@ -50,17 +53,66 @@ public void obeyClaimBatchMaxSize()throws IOException {
...
+
+        /**
+         * Origin.
+         */
+        private final BatchClaims origin;
+
+        /**
+         * Count of {@link BatchClaims#close()} invocations.
+         */
+        private int closecount;
```
Comment (krzyk) [inline src/test/java/com/zerocracy/claims/BatchClaimsTest.java]: @paulodamaso compound names should be avoided, here `count` is a good fit

File: src/test/java/com/zerocracy/claims/BatchClaimsTest.java [hunk 2]
```diff
@@ -50,17 +53,66 @@ public void obeyClaimBatchMaxSize()throws IOException {
         for (int idx = 0; idx < count; ++idx) {
             claims.add(new ClaimOut());
         }
-        final BatchClaims batch = Mockito.mock(BatchClaims.class);
-        Mockito.doCallRealMethod().when(batch).maximum();
```
Comment (krzyk) [inline src/test/java/com/zerocracy/claims/BatchClaimsTest.java]: @paulodamaso please also remove file `src/test/resources/mockito-extensions/org.mockito.plugins.MockMaker`, as it is not needed (and we don't want to encourage final classes mocking)

**Comment by krzyk** [comment]
@paulodamaso thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@g4s8 ping

**Comment by g4s8** [comment]
@rultor merge

**Comment by paulodamaso** [comment]
@g4s8 Strange this error happening in rultor but not in travis.

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1632](https://github.com/zerocracy/farm/issues/1632) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@krzyk According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three major problems while future reviews.

**Comment by krzyk** [comment]
@ypshenychka I confirm

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**